bodhi/server/models.py

```diff
@@ -789,6 +789,14 @@ class Update(Base):
     user_id = Column(Integer, ForeignKey('users.id'))
 
     @property
+    def mandatory_days_in_testing(self):
+        if self.critpath:
+            return int(config.get('critpath.stable_after_days_without_negative_karma'))
+
+        days = self.release.mandatory_days_in_testing
+        return days if days else 0
```

Comment:
Btw, there's another path in that code that would let you set only one config item if you want to do that instead - it involves setting ```{id_prefix}.mandatory_days_in_testing``` in the config. Slightly simpler and it'll achieve the same result. Either way you prefer is fine of course ☺